.gitignore

```diff
@@ -0,0 +1,8 @@
+**/__pycache__/
```

Comment:
can we drop this file?

Reply:
Sure, I pulled it out into a separate PR: https://github.com/awslabs/Renate/pull/190